otter/integration/tests/test_lbsh.py

```diff
@@ -34,6 +38,7 @@ class TestLoadBalancerSelfHealing(unittest.TestCase):
     This class contains test cases to test the load balancer healing
     function of the Otter Converger.
     """
+    timeout = 1800
```

Comment:
This is used to time out the a test specifically.  If any of the test methods in this class runs for more than 1800 seconds, trial will kill it.  Otherwise the default is something much shorter (which works against mimic, but not against production).

Everywhere is to time out polling for stuff, like a specific `wait_for_X` operation that we wrote ourselves.  I figured since a test includes multiple `wait_for_X` with the default timeouts, then the test timeout should probably be a little longer.